### ISSUE TITLE
Symmetry Feedback: Table 15.2 Too Wide

### ISSUE THREAD

**Issue by openstaxalina** [openstax/cnx-recipes#1956]
# Scenario

I don't think this is art...I think the code is styling the table this wide.
![Screen Shot 2020-02-24 at 1 56 27 PM](https://user-images.githubusercontent.com/7863733/75186520-a4d78300-570d-11ea-861f-f461597397b3.png)

# Acceptance Criteria (required)

# Test Plan

**Comment by ajluk** [comment]
These is caused by `table-layout: fixed;` property I have added to tables in exercises in both Biologies (Biology 2e, AP Biology) in this PR: https://github.com/openstax/cnx-recipes/pull/1676, because tables in AP Biology look bad in some places (text was not centered in the cells.) - see this comment for realted issue: https://github.com/openstax/cnx-recipes/issues/1675#issuecomment-565347522. I'm not sure how can it be resolved, because removing it will affect tables in AP Biology exercises as well. This screenshot shows how it will look after removing it in Biology 2e: 
![image](https://user-images.githubusercontent.com/55203598/75246058-44a91580-57cf-11ea-8792-9ad5be7532c9.png)

**Comment by lwtchu** [comment]
@ajluk what does `table-layout: fixed;` mean?

**Comment by ajluk** [comment]
@lwtchu It "sets a fixed table layout algorithm. The table and column widths are set by the widths of table and col or by the width of the first row of cells. Cells in other rows do not affect column widths. If no widths are present on the first row, the column widths are divided equally across the table, regardless of content inside the cells", so in most of the cases it based on the first row. Set the width of those, and the rest of the table follows. I have set it to resolve tables issue in two column layout issue in AP Biology. They looked like this before it was set: 
![image](https://user-images.githubusercontent.com/55203598/75425893-de91cf00-5943-11ea-913a-fc0a16b01c2a.png)
 They were almost touching the other column, so I have set width to 100% for them, but then it caused another issue:
![image](https://user-images.githubusercontent.com/55203598/75425662-72af6680-5943-11ea-9931-e5539f2166ea.png)
In some cells the space has been reduced. It is when I have added the `table-layout: fixed;` , because I haven't found another solution.
